docs/guides/transportation.mdx

```diff
@@ -49,6 +63,210 @@ are currently the most developed part of the `transportation` schema. Read more
+| prohibited_transitions | STRUCT[] | Defines where traveling from the segment to another is disallowed for navigation. This covers things situations prohibited turns or a transition from road to bike lane disallowing cars. |
+| road_surface | STRUCT[] | Defines the surface material on a road such as paved, asphalt, or unpaved. |
+| road_flags | STRUCT[] | Additional properties relevant to roads such as is_bridge or is_under_construction. |
+| speed_limits | STRUCT[] | Defines the speed limit of the road segment. |
+| width_rules | STRUCT[] | Defines the width of the road segment for rendering. |
+</TabItem>
+<TabItem value="connector" label="connector" default>
+| column_name | column_type | Description |
+|---|---|---|
+| id | VARCHAR | A feature ID. This may be an ID associated with the Global Entity Reference System (GERS) if—and-only-if the feature represents an entity that is part of GERS. |
+| geometry | BLOB | The line representation of the segment's location. Segment's geometry which MUST be a LineSting as defined by GeoJSON schema. |
+| bbox | STRUCT | Area defined by two longitudes and two latitudes: latitude is a decimal number between -90.0 and 90.0; longitude is a decimal number between -180.0 and 180.0. |
+| version | INTEGER | Version number of the feature, incremented in each Overture release where the geometry or attributes of this feature changed. |
+| sources | STRUCT[] | The array of source information for the properties of a given feature, with each entry being a source object which lists the property in JSON Pointer notation and the dataset that specific value came from. All features must have a root level source which is the default source if a specific property's source is not specified. |
+</TabItem>
+</Tabs>
+</div>
+</details>
+
+### Subtypes
+Transportation segments are divided into three subtypes: **rail**, **water**, and **road**. The road subtype is then further divided into a variety of different classes based on usage captured in the table below.
+
+<details>
+    <summary>Class names and feature counts</summary>
+<div>
+<Tabs>
+<TabItem value="classes" label="classes" default>
+| subtype | class | Feature count, July 2024 release |
+|---|---|---|
+| rail |  | 1,891,720 |
+| road | alley | 1,507,160 |
+| road | bridleway | 94,519 |
+| road | crosswalk | 1,736,678 |
+| road | cycleway | 1,213,800 |
+| road | driveway | 14,722,896 |
+| road | footway | 14,149,673 |
+| road | living_street | 2,960,512 |
+| road | motorway | 1,047,400 |
+| road | parking_aisle | 5,860,995 |
+| road | path | 12,487,665 |
+| road | pedestrian | 445,687 |
+| road | primary | 6,917,142 |
+| road | residential | 123,084,186 |
+| road | secondary | 10,719,092 |
+| road | sidewalk | 2,786,910 |
+| road | steps | 1,643,010 |
+| road | tertiary | 19,288,033 |
+| road | track | 23,592,666 |
+| road | trunk | 3,843,984 |
+| road | unclassified | 28,552,184 |
+| road | unknown | 31,725,341 |
```

Comment:
These will change quite a bit this release and we may want to include a further breakdown using subclass?